src/components/Statistics/Statistics.tsx

```diff
@@ -1,44 +1,41 @@
-import Graph from '@components/Graph/Graph';
-import { Graphs } from '@components/Graph/Graph.style';
+import Graph from '@components/Graph';
 import Slice from '@components/Slice';
 import TextList from '@components/TextList';
 import { dataNumberFcu } from '@data';
 import { useMemo } from 'react';
 import useSWR from 'swr';
 import Band from './Band';
-import { Container } from './Statistics.style';
-
-const monthToString = [
-  'Jan',
-  'Fev',
-  'Mar',
-  'Avr',
-  'Mai',
-  'Juin',
-  'Juil',
-  'Aout',
-  'Sept',
-  'Oct',
-  'Nov',
-  'Dec',
-];
+import { Container, GraphsWrapper } from './Statistics.style';
+
+type NbVisitType = { nb_visits?: number | string };
```

Comment:
try to be consistent, you don't use Type in the other type name. The CamelCase should be enough